docs/tutorials/acme/quick-start/index.rst

```diff
@@ -488,9 +488,13 @@ Create this definition locally and update the email address to your own. This
 email required by Let's Encrypt and used to notify you of certificate
 expirations and updates.
 
+- staging issuer: `staging-issuer.yaml`_
+
 .. literalinclude:: example/staging-issuer.yaml
    :language: yaml
 
+.. _`staging-issuer.yaml`: https://raw.githubusercontent.com/jetstack/cert-manager/master/docs/tutorials/acme/quick-start/example/staging-issuer.yaml
```

Comment:
Can you update references to `master` with `release-0.7` here and below? 😄

Reply:
Done :)

Reply:
Although I am still hitting this:
```
I0307 12:01:59.689206       1 http.go:111] could not reach 'http://192.171.139.249.xip.io/.well-known/acme-challenge/DyAo_CkVHB0LU54rCbN0CV12YnK0NhLeUI_CNWRwPWc': failed to GET 'http://192.171.139.249.xip.io/.well-known/acme-challenge/DyAo_CkVHB0LU54rCbN0CV12YnK0NhLeUI_CNWRwPWc': Get http://192.171.139.249.xip.io/.well-known/acme-challenge/DyAo_CkVHB0LU54rCbN0CV12YnK0NhLeUI_CNWRwPWc: dial tcp 192.171.139.249:80: connect: connection timed out
```